src/handlers/feature_handlers.cpp

```diff
@@ -6449,6 +6575,9 @@ void aiVehicleInstanceFeatureHandler::Draw(int lod)
     }
 
     inst->setLOD(lod + 1);
+
+    if (BackwardsCheat)
```

Comment:
Why are you rotating the matrix at the end of the function?

Reply:
Check MM1's aiVehicleInstance::Draw function, they wrote the cheat like this.